scripts/templates/docker-compose.yml

```diff
@@ -93,6 +94,16 @@ services:
       - wk-oidc-server
     networks:
       - ${NETWORKS}
+##BEGIN drawio
```

Comment:
This part might be unnecessary. Since your configuration do not show any integration with drawio